nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -95,23 +89,23 @@ if( workflow.profile == 'awsbatch') {
 }
 
 // Stage config files
-ch_multiqc_config = Channel.fromPath(params.multiqc_config)
-ch_output_docs = Channel.fromPath("$baseDir/docs/output.md")
+ch_multiqc_config = file(params.multiqc_config, checkIfExists: true)
+ch_output_docs = file("$baseDir/docs/output.md", checkIfExists: true)
```

Comment:
```suggestion
ch_output_docs = Channel.fromPath("$baseDir/docs/output.md", checkIfExists: true)
```

Reply:
Ill revert it back 👍 To be honest, Im still confused why this was breaking things on AWS. When I was in Tubingen, Paolo insisted that I should be using `file()` instead. Maybe we need to create an issue for this somewhere so at least I can refer to it 😅

Reply:
In `DSL1` one of the biggest advantages of using `file` here is that it becomes a value channel and so can be reused without having to declare numerous named channels e.g.
https://github.com/nf-core/chipseq/blob/333bba334512625775157aa0087c023cf0bde397/main.nf#L164-L170

instead of:
https://github.com/nf-core/atacseq/blob/c8d57b5d8e0a2715624ada38803646ed2c97cc67/main.nf#L170-L178

Reply:
Sure, but didn't using `file` break stuff? You removed them at the bottom of this PR..

Reply:
I remember the exact same conversation with Paolo in Tübingen at the hackathon too - he insisted on having `file(...)` for these instead of channels but I also had it somewhere back in my head that this caused trouble on AWS ...

Reply:
It did but I thinking back it may have been because it was a directory and not an actual file. I think...would need to test a little more. You mean using `file()` in the `workflow.OnComplete` block?

Reply:
Ok. Keeping `file()` then. Speak now or hold your peace until the next release 👍

Reply:
If we really encounter a big problem, we can still do a mini bugfix with 1.7.1